AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/MutationSync/OutgoingMutationQueue/OutgoingMutationQueue.swift

```diff
@@ -222,6 +222,21 @@ final class OutgoingMutationQueue: OutgoingMutationQueueBehavior {
         }
     }
 
+    private func processSuccessEvent(_ mutationEvent: MutationEvent,
+                                     mutationSyncMetadata: MutationSync<AnyModel>?) {
+        if let mutationSyncMetadata = mutationSyncMetadata {
+            MutationEvent.updatePendingMutationEventVersionIfNil(
+                for: mutationEvent.modelId,
+                mutationSync: mutationSyncMetadata,
+                storageAdapter: storageAdapter) { result in
+                self.log.verbose("\(#function) received result after syncing version from API: \(result)")
```

Comment:
this log line seems to be misleading since syncing the version from the API result is contingent on the logic in `updatePendingMutationEventVersionIfNil`, it may or may not have occurred. I don't think a log is necessary here